tools/data-api-differ/internal/differ/differ_api_resource.go

```diff
@@ -48,7 +48,11 @@ func (d differ) changesForApiResource(serviceName, apiVersion, apiResource strin
 			ApiVersion:   apiVersion,
 			ResourceName: apiResource,
 		})
-		// intentionally not returning here
+		if !includeNestedChangesWhenNew {
+			return &output, nil
+		}
+		// Otherwise we intentionally don't returning here to return the full output - needed to detect any new
```

Comment:
grammar in the comment is off, I don't know what this is supposed to be saying

Reply:
Good shout - updated.